requirements.txt

```diff
@@ -28,4 +28,5 @@ scikit-image==0.24.0
 typer==0.12.5
 transformers==4.45.2
 torch-geometric==2.6.1
-mlflow==2.17.2
\ No newline at end of file
+tornado==6.4.1
```

Comment:
Please do not commit tornado, that should be adressed in this PR https://github.com/meteofrance/py4cast/pull/104